package.json

```diff
@@ -20,7 +20,7 @@
     "storybook": "start-storybook -p 9009 -c .storybook",
     "test:a11y": "echo 'Axe will run on this command'",
     "test:debug": "node --inspect node_modules/.bin/jest --runInBand",
-    "test": "jest --passWithNoTests",
+    "test": "TZ=UTC jest --passWithNoTests",
```

Comment:
Can we remove `TZ=UTC`? We have changed DatePicker component and there is no need to specify timezone here.